root/account/registered.tt

```diff
@@ -1,26 +0,0 @@
-[% WRAPPER 'layout.tt' title=l("Account Created") %]
```

Comment:
This was unused so I guess it can be removed. But I reused some of the email-verification text below.